superset/assets/spec/javascripts/explorev2/utils_spec.jsx

```diff
@@ -0,0 +1,56 @@
+import { it, describe } from 'mocha';
+import { expect } from 'chai';
+import URI from 'urijs';
+import { getExploreUrl } from '../../../javascripts/explorev2/exploreUtils.js';
+
+describe('utils', () => {
+  const formData = {
+    datasource: '1__table',
+  };
+  const sFormData = JSON.stringify(formData);
+  function compareURI(uri1, uri2) {
+    expect(uri1.toString()).to.equal(uri2.toString());
+  }
+
+  it('getExploreUrl generates proper base url', () => {
+    expect(location.href).to.equal('about:blank');
```

Comment:
Oh I left it there for peopler reading this to understand that this is the value of `location.href` in the test context, and also to know that there is this expectation if that ever changes. I thought about messing with the mocha configuration and I feel it may still happen in the future...